### PR TITLE
feat: new role (granted to execute explain) [2.38-DHIS2-12605]

### PR DESCRIPTION
**Pull request by d-bernat**
There is a new role needed. User in the role can perform explain analyze query on analytics queries: 
http://localhost:8080/dhis/api/38/analytics/explain?dimension=dx%3ATcTRasl29wo,pe%3ALAST_12_MONTHS&filter=ou%3AUSER_ORGUNIT&includeNumDen=false&skipMeta=true&skipData=false

### CODE REVIEW

File: dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/v38/V2_38_32__Add_new_user_role_for_analytics_explain_endpoint.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                superUserRoleIds.add( superUserRole.getInt( 1 ) );
+            }
+        }
+        catch ( SQLException e )
+        {
+            log.error( "Flyway java migration error", e );
+            throw new FlywayException( e );
+        }
+
+        if ( !superUserRoleIds.isEmpty() )
```
Comment (gnespolino) [inline dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/v38/V2_38_32__Add_new_user_role_for_analytics_explain_endpoint.java:69]: why we add F_PERFORM_ANALYTICS_EXPLAIN only when superUser has no roles assigned ?

File: dhis-2/dhis-services/dhis-service-core/src/main/resources/i18n_global.properties [hunk 2]
```diff
@@ -1801,7 +1802,6 @@ oauth2_clients=OAuth2 clients
 
 #-- Authorities ---------------------------------------------------------------#
 
-F_TRACKED_ENTITY_INSTANCE_SEARCH_IN_ALL_ORGUNITS=Search tracked entity instance across all organisation units
```
Comment (gnespolino) [inline dhis-2/dhis-services/dhis-service-core/src/main/resources/i18n_global.properties:1804]: why we removed this one ?

**Review by gnespolino — APPROVED** [review]
Approved with a couple of questions I don't understand! Good one!

File: dhis-2/dhis-services/dhis-service-core/src/main/resources/i18n_global.properties (hunk 2 above)
Comment (d-bernat) [reply dhis-2/dhis-services/dhis-service-core/src/main/resources/i18n_global.properties:1804]: it was twice there, I took the latest one written by Lars...

File: dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/v38/V2_38_32__Add_new_user_role_for_analytics_explain_endpoint.java (hunk 1 above)
Comment (d-bernat) [reply dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/v38/V2_38_32__Add_new_user_role_for_analytics_explain_endpoint.java:69]: ???

**Review by maikelarabori — APPROVED** [review]
Looks good @d-bernat, just a very minor comment.

**Pull request merged**